Busobozihakiim/EPICMAIL_api
# Ft view all emails #164691565

### Users should be able to view their inbox 

#### What does this PR do?
- Enables users to view all the messages they have in their inbox

#### Description of Task to be completed?
- Write the /message endpoint to implement the feature
- Write test for this feature.

#### How should this be manually tested?
- Use this link in your postman app with method POST
- https://epicmail007.herokuapp.com/api/v1/messages
- You should get the message "You dont have messages currently"
- Proceed and create a message using the same link but with the HTTP method POST
- visit the link again but with GET method, you should see a response of the format
{
     'id' : integer , 
     'createdOn': Datetime ,
     'subject': String ,
     'message': String ,
     'senderId': String ,
     'receiverId':String ,
     'status' : String
}
#### Relevant Pivotal Tracker Stories
#164691327
